style.scss

```diff
@@ -0,0 +1,87 @@
+//declaring values in sass
+$primary-dark-color: gray;
+$primary-bright-color: lightgray;
+$onePercent : 1%;
+//using mixin, just to demonstrate
+@mixin setMinimums($height, $width){
+        min-height: $height;
+        min-width: $width;
+}
+@mixin solidBlackBorder($width){
+    border: black solid $width;
+}
+//and this feature :)
+%text-shared{
+    text-align: center;
+    font-weight: bold;
+}
+
+code {
+    font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
+      monospace;
+  }
+
+body{
+    background-color: $primary-dark-color;
+    margin: 0;
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+    sans-serif;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+}
+
+.main--box{
+    width: 45%;
+    margin:auto;
+    background-color: $primary-bright-color;
+    @include solidBlackBorder(2px);
+    border-radius: 5px;
+  }
+  
+  .box--row{
+    display: flex; 
+  }
+  
+  .box--field{
+    //include font styling
+    @extend %text-shared;
+    //include mixin for height and width
+    @include setMinimums(110px, 33.33%);
+    //and for borders
+    @include solidBlackBorder(1px);
+    //do nesting with sass
+   // border: {
+    //    style: solid;
+    //    width: 1px;
+    //    color: black;
+   // }
+    //and use & to change pointer while we hover
+    &:hover{
+        cursor:pointer;
+    }
+    box-sizing: border-box;
+    line-height: 100px;
+    font-size: 50px;
+    text-shadow: 0r, 5px $primary-bright-color,
+    0 0 10px $primary-bright-color,
+    0 0 20px $primary-dark-color, 
+    0 0 30px $primary-dark-color, 
+    0 0 40px $primary-dark-color, 
+    0 0 55px $primary-dark-color, 
+    0 0 70px $primary-dark-color; 
+  }
+  
+  .game--info, h1, button{
+    //include font styling here also!
+    @extend %text-shared;
+    // and use sass values defined at start
+    padding: $onePercent;
+    margin: $onePercent;
+  }
+
+  @media only screen and (max-width: 780px) {

```

Comment:
check common breakpoints for devices (1200px, 1024px, 768px and similar), and create variables for them, and if using sass, you can make mixins for responsive design